<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPrice</class>
 <widget class="QDialog" name="AddPrice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>466</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Adding New Price</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="m_lDate">
       <property name="text">
        <string>Today Date :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="m_deDate">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="m_lCategory">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Category :</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1,1">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QComboBox" name="m_cbCategory">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_tbNewCategory">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="expenses.qrc">
         <normaloff>:/eXpenses/Resources/add.png</normaloff>:/eXpenses/Resources/add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_tbDeleteCategory">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="expenses.qrc">
         <normaloff>:/eXpenses/Resources/remove.png</normaloff>:/eXpenses/Resources/remove.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="m_lProduct">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Product :</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,1,1">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QComboBox" name="m_cbProduct">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_tbNewProduct">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="expenses.qrc">
         <normaloff>:/eXpenses/Resources/add.png</normaloff>:/eXpenses/Resources/add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_tbDeleteProduct">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="expenses.qrc">
         <normaloff>:/eXpenses/Resources/remove.png</normaloff>:/eXpenses/Resources/remove.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="m_lOldPrice">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Old Price for Unit :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="m_dsbOldPrice">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <double>999999.989999999990687</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_lNewPrice">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>New Price for Unit :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="m_dsbNewPrice">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <double>999999.989999999990687</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="m_pbAddPrice">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Add Price</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="expenses.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_pbAddPrice</sender>
   <signal>clicked()</signal>
   <receiver>AddPrice</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_cbCategory</sender>
   <signal>activated(QString)</signal>
   <receiver>AddPrice</receiver>
   <slot>onCategoryChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_cbProduct</sender>
   <signal>activated(QString)</signal>
   <receiver>AddPrice</receiver>
   <slot>onProductChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_tbNewCategory</sender>
   <signal>clicked()</signal>
   <receiver>AddPrice</receiver>
   <slot>onNewCategory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>632</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_tbNewProduct</sender>
   <signal>clicked()</signal>
   <receiver>AddPrice</receiver>
   <slot>onNewProduct()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>632</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>AddPrice</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>572</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_tbDeleteCategory</sender>
   <signal>clicked()</signal>
   <receiver>AddPrice</receiver>
   <slot>onDeleteCategory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>789</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_tbDeleteProduct</sender>
   <signal>clicked()</signal>
   <receiver>AddPrice</receiver>
   <slot>onDeleteProduct()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>789</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onCategoryChanged(QString)</slot>
  <slot>onProductChanged(QString)</slot>
  <slot>onNewCategory()</slot>
  <slot>onNewProduct()</slot>
  <slot>onDeleteCategory()</slot>
  <slot>onDeleteProduct()</slot>
 </slots>
</ui>
